### DISCUSSION TITLE
DuckDB exports load.sql with an absolute path. Exporting with relative paths would make it easier to use in other machines.

### DISCUSSION THREAD

**Discussion by tanin47** [duckdb/duckdb#11943] [Ideas]
I was trying to export the database, zip it, and send to my friend. It turns out my friends can't import it due to the COPY statement using an absolute path on my machine.

**Comment by Tishj** [comment]
It's been a while since I made this PR
<https://github.com/duckdb/duckdb/pull/5965>

What part is not relative?

**Comment by tanin47** [comment]
`load.sql` contains the full paths for me.

It might come from this line: https://github.com/duckdb/duckdb/pull/5965/files#diff-b0890d1f1861a054495b8f7ed604fd0691efd69a9c62a5feb8fb40f653cc55f6R101

**Comment by tanin47** [comment]
I've looked at it more. While `load.sql` contains absolute paths, those absolute paths don't seem to do anything.

The problem that I'm having comes from this bug instead: https://github.com/ZJONSSON/node-unzipper/issues/261

I'll close this feature request.
